java/src/main/java/org/synbiohub/PrepareSubmissionJob.java

```diff
@@ -482,8 +484,14 @@ public void visit(Identified identified, TopLevel topLevel) {
 						SBOLDocument tlDoc;
 						try {
 							tlDoc = sbh.getSBOL(URI.create(topLevelUri));
-						} catch (SynBioHubException e) {
-							tlDoc = null;
+						} catch (SynBioHubException e1) {
+							StringWriter sw = new StringWriter();
```

Comment:
You can't `printStackTrace(System.err)` ?

Reply:
Probably, but I was also converting to a string to return in errorLog.